src/prediction.jl

```diff
@@ -249,4 +249,68 @@ function build_future_df(base_df, to_predict::Symbol)
 
     return(reduce(vcat, result_lines))
 
-end # end function1
\ No newline at end of file
+end # end function1
+
+function tryparsecol(Type::DataType, col)
+    try
```

Comment:
Let's try to avoid using exception handling for control flow. This isn't python!

`tryparse()` from Base returns `nothing` if it doesn't work, we can do the same here. Or, if you really want to return the col to avoid an extra step, something like

```
julia> function tryparsecol(T, col)
           newcol = tryparse.(T, col)
           return any(isnothing, newcol) ? col : newcol
       end
tryparsecol (generic function with 1 method)

julia> tryparsecol(Int, ["1", "1.3"])
2-element Vector{String}:
 "1"
 "1.3"

julia> tryparsecol(Int, ["1", "2"])
2-element Vector{Int64}:
 1
 2
```